whipper/common/mbngs.py

```diff
@@ -250,11 +251,13 @@ def _getMetadata(release, discid=None, country=None):
                     track.artist = trackCredit.getName()
                     track.sortName = trackCredit.getSortName()
                     track.mbidArtist = trackCredit.getIds()
+                    track.recordingArtist = recordingCredit.getName()
 
                     track.title = t.get('title', t['recording']['title'])
                     track.mbid = t['id']
                     track.mbidRecording = t['recording']['id']
                     track.mbidWorks = _getWorks(t['recording'])
+                    track.isClassical = (track.artist != track.recordingArtist)
```

Comment:
This is a bad check. Lots of recordings will not have the same Track Artist as their Recording Artist. E.g., lots of "ArtistA feat. ArtistB" Recordings will have something like "ArtistA ft. ArtistB" or even just "ArtistA" in their Track Artists. Not to mention cases such as https://musicbrainz.org/edit/66442558 - if you absolutely want an automated check, check for the "classical" genre/tag on Release or Release Group. Otherwise just rely on people using the `--classical` flag when ripping classical releases instead of trying to automatically discover when something is classical.

Reply:
I will follow the latter part: "rely on people using the --classical flag when ripping classical releases instead of trying to automatically discover when something is classical."